core/src/main/java/jenkins/tools/GlobalToolConfiguration.java

```diff
@@ -81,12 +80,12 @@ public synchronized void doConfigure(StaplerRequest req, StaplerResponse rsp) th
         FormApply.success(req.getContextPath() + "/manage").generateResponse(req, rsp, null);
     }
 
-    private boolean configure(StaplerRequest req, JSONObject json) throws hudson.model.Descriptor.FormException, IOException {
+    private boolean configure(StaplerRequest req, JSONObject json) throws Descriptor.FormException, IOException {
         Jenkins j = Jenkins.get();
         j.checkPermission(Jenkins.ADMINISTER);
 
         boolean result = true;
-        for(Descriptor<?> d : Functions.getSortedDescriptorsForGlobalConfig(FILTER)){
+        for (Descriptor<?> d : Functions.getSortedDescriptorsForGlobalConfigByDescriptor(FILTER)) {
```

Comment:
(Note to self: This change makes sense since the old method is being deprecated.)